### ISSUE TITLE
Update material library to 1.1.0

### ISSUE THREAD

**Issue by nbradbury** [woocommerce/woocommerce-android#1910]
We're currently using v1.0.0 of the material library, but v1.1.0 is the latest stable release. I was going to update it as part of #1909, but that resulted in thw following `WCMaterialOutlinedEditTextView` crash when tapping "Inventory" in product detail:

```
IllegalArgumentException: The style on this component requires your app theme to be Theme.MaterialComponents (or a descendant).
```

**Comment by nbradbury** [comment]
Note: I added this to the "Dark Mode" project since that resolves the crash (since dark mode uses the material theme).

**Comment by nbradbury** [comment]
Closed via #1917
